README.md

```diff
@@ -102,49 +102,455 @@ Here we will highlight a few of the capabilities of the Greentea command line in
 We will assume for the examples below that the above `test_spec.json` file is in current directory.
 
 ### Listing all tests
-We can use the `-l` arguement to list all availble tests:
+We can use the `-l` argument to list all availble tests:
 
 ```
 $ mbedgt -l
-mbedgt: using 'test_spec.json' from current directory!
-mbedgt: available tests for built 'K64F-ARM', location './BUILD/K64F/ARM'
+mbedgt: greentea test automation tool ver. 1.2.5
+mbedgt: using multiple test specifications from current directory!
+        using 'BUILD\tests\K64F\ARM\test_spec.json'
+        using 'BUILD\tests\K64F\GCC_ARM\test_spec.json'
+mbedgt: available tests for built 'K64F-GCC_ARM', location 'BUILD/tests/K64F/GCC_ARM'
         test 'tests-mbedmicro-rtos-mbed-mail'
-        test 'tests-mbed-drivers-c_strings'
-mbedgt: available tests for built 'K64F-GCC', location './BUILD/K64F/GCC_ARM'
+mbedgt: available tests for built 'K64F-ARM', location 'BUILD/tests/K64F/ARM'
```

Comment:
Yes, this should be fixed in Greentea's source (I copied this straight from the command line).